test/libyul/YulOptimizerTest.cpp

```diff
@@ -259,8 +263,20 @@ bool YulOptimizerTest::run(ostream& _stream, string const& _linePrefix, bool con
 		return false;
 	}
 
-	m_obtainedResult = m_optimizerStep + "\n" + AsmPrinter{m_yul}(*m_ast) + "\n";
+	m_obtainedResult = AsmPrinter{m_yul}(*m_ast) + "\n";
 
+	bool success = true;
+	if (m_optimizerStep != m_validatedSettings["step"])
+	{
+		string nextIndentLevel = _linePrefix + "  ";
+		AnsiColorized(_stream, _formatted, {formatting::BOLD, formatting::CYAN}) <<
+			_linePrefix <<
+			"Invalid optimizer step (should be " <<
```

Comment:
It's probably always better to not only output what "should be", but also what "is" - that way it would probably have been clearer why the tests are failing :-). Maybe even makes sense to quote it to account for whitespace mismatches.